src/components/Graph.js

```diff
@@ -0,0 +1,67 @@
+import { Paper } from '@material-ui/core'
+import { Bar, Doughnut, Pie } from 'react-chartjs-2'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+const chartOptions = {
```

Comment:
These options may only be applicable to `Bar` components. Would you mind creating a new const that stores both `Pie` and `Doughnut` as well? 🙃

Reply:
Done. I've created just a simple deafult options, but you can change it using props.